nodes/cma/nodes-cma-autoscaling-custom-trigger-auth.adoc

```diff
@@ -47,18 +67,42 @@ metadata: <1>
   name: secret-cluster-triggerauthentication
 spec:
   secretTargetRef: <2>
-  - parameter: user-name <3>
+  - parameter: username <3>
     name: secret-name <4>
-    key: USER_NAME <5>
-  - parameter: user-password
+    key: username <5>
+  - parameter: password
     name: secret-name
-    key: USER_PASSWORD
+    key: password
 ----
 <1> Note that no namespace is used with a cluster trigger authentication.
-<2> Specifies that this trigger authentication uses a secret for authorization.
+<2> Specifies that this trigger authentication uses a secret for authorization when connecting to the metrics endpoint.
+<3> Specifies the authentication parameter to supply by using the secret.
+<4> Specifies the name of the secret to use.
+<5> Specifies the key in the secret to use with the specified parameter.
+
+.Example trigger authentication using a secret for CA details
+[source,yaml]
+----
+kind: TriggerAuthentication
+apiVersion: keda.sh/v1alpha1
+metadata:
+  name: secret-triggerauthentication
+  namespace: my-namespace <1>
+spec:
+  secretTargetRef: <2>
+    - parameter: bearerToken <3>
+      name: my-secret <4>
+      key: bearerToken <5>
+    - parameter: ca <6>
+      name: my-secret
+      key: ca.crt
```

Comment:
I believe this needs to be `ca` not `ca.crt`. The `parameter` field refers to the scaled object parameter, which must always be `ca` (or CMA won't understand what to do with it). So that one is correct. The `key` field tells CMA which field it should look for in `my-secret`, so it must match the field name in the secret. So either `key` here needs to be `ca` or the `ca: "ca"` in the secret (currently line 29) would need to become `ca.crt: "ca"` (either way is valid as long as the secret field name and `key`'s value match).

Reply:
@joelsmith In the secret, what would actually be used as these values? 

```
data:
  cert: "my-actual-cert"
  key: "my-cert-private-key"
  ca: "ca" 
```
> The parameter field refers to the scaled object parameter,

What does this mean? Do we need to define it, or just say "Specify `ca` to use a certificate?

Reply:
The data would look something like the cert and key examples you mentioned, but the ca field would also look like those. It might be good to show what it actually might look like.

Reply:
Here's a more complete example:
```
$ oc create secret generic my-secret --from-file=key=client-key.pem --from-file=cert=client-cert.pem --from-file=ca=server-ca-cert.pem
```
Which results in a secret like
```
apiVersion: v1
kind: Secret
metadata:
  name: my-secret
  namespace: default
type: Opaque
data:
  ca: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0S0tCk1JSUdE...
  cert: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSU...
  key: LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tk1JSUp...
```